src/Properties/ModelPropertyExtension.php

```diff
@@ -276,4 +279,29 @@ private function castPropertiesType(Model $modelInstance): void
             $this->tables[$modelInstance->getTable()]->columns[$name]->writeableType = $realType;
         }
     }
+
+    private function hasAttribute(ClassReflection $classReflection, string $propertyName): bool
+    {
+        if ($classReflection->hasNativeMethod('get'.Str::studly($propertyName).'Attribute')) {
+            return true;
+        }
+
+        $camelCase = Str::camel($propertyName);
+
+        if ($classReflection->hasNativeMethod($camelCase)) {
+            $methodReflection = $classReflection->getNativeMethod($camelCase);
+
+            $returnType = ParametersAcceptorSelector::selectSingle($methodReflection->getVariants())->getReturnType();
+
+            if ($methodReflection->isPublic()) {
+                return false;
+            }
+
+            if (! (new ObjectType(Attribute::class))->isSuperTypeOf($returnType)->yes()) {
+                return false;
+            }
+        }
```

Comment:
@canvural Why have this block? The returned value is `false` anyways.

Reply:
@szepeviktor Oh, thanks! It should  return true at the end, not false.

Reply:
You're welcome.